DOL.WHD.Section14c.Web/src/modules/components/mainHeaderControl/mainHeaderControlDirective.test.js

```diff
@@ -0,0 +1,18 @@
+describe('mainHeaderControl', function() {
+
+    beforeEach(module('14c'));
+
+    var element, rootScope;
+    beforeEach(function () {
+        element = angular.element('<main-header-control/>');
+    	inject(function ($rootScope, $compile) {
+            rootScope = $rootScope;
+			//$compile(element)(rootScope);
+			rootScope.$digest();
+        });
+    });
+
+    it('invoke directive', function() {
+        // compiled in init.
```

Comment:
updated directive unit tests to assert it was created https://github.com/AppliedIS/dol-whd-14c/commit/e73d8d8cc8291dbbcf47a7da435a7ad963cc54a4